examples/model_predictive_controllers/pmsm_mpc_dq_current_control.ipynb

```diff
@@ -1110,9 +343,9 @@
    ],
    "source": [
     "%matplotlib notebook\n",
-    "\n",
+    "visu = MotorDashboard(state_plots=['i_sq', 'i_sd', 'u_sq', 'u_sd'], update_interval=10), # visualization\n",
     "env = gem.make('AbcCont-CC-PMSM-v0',\n",
-    "            visualization=MotorDashboard(state_plots=['i_sq', 'i_sd', 'u_sq', 'u_sd'], update_interval=10), # visualization\n",
+    "            visualization=visu\n",
```

Comment:
visu,

Reply:
line 360: u_sup has to be changed to u_nominal for the actual version of gem.

Reply:
Info: When I start the last two cells again, the reference generators don't work correct. I guess, the generators are using a default value in this case. This is probably a new issue/enhancement.

![image](https://user-images.githubusercontent.com/95769661/151197698-11acb170-607e-4a6a-94dd-50c25a8a49c0.png)

Reply:
That is no error, just a StepReference Generator. This environment switches the reference generator every n-steps between a sinusoidal, triangular and step reference.

Reply:
Yes, under these circumstances, I can reproduce the error. I was also able to identify the source: A new env. is created but the reference generators are not instantiated again. So, you use the same instances as before. Somehow, this fails. 
If you run also the third to last cell and create new ref gens, everything works fine. 

At first sight, I was not able to locate the error. However, I will fix this soon within another PR (as you already mentioned)